frontend/reducers/ui_reducers/song_card_dropdown_reducer.js

```diff
@@ -3,7 +3,7 @@ import { SONGCARD_DROPDOWN_OPEN,
 } from "../../actions/ui_actions";
 
 
-const songCardDropdownReducer = (songCardDropdownState = {}, action) => {
+const songCardDropdownReducer = (songCardDropdownState = {isOpen: false}, action) => {
```

Comment:
Do we need the default value here?

Reply:
Yes, default `isOpen: false` is required so that the dropdown isn't visible when we first load the component.